<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <include file="$(find rr_iarrc)/launch/startlight_watcher.launch"/>

    <node name="urc_controller" pkg="rr_iarrc" type="urc_controller" output="screen">
        <param name="startSignal" type="string" value="/start_detected"/>
        <param name="resetSignal" type="string" value="/reset_detected"/>
        <param name="turn_speed" type="double" value="0.60"/>
        <param name="left_turn_steering" type="double" value="-0.20"/>
        <param name="right_turn_steering" type="double" value="0.20"/>
        <param name="left_offset" type="double" value="0.0"/>
        <param name="right_offset" type="double" value="0.0"/>
       	<param name="forward_timeout" type="double" value="10.0"/>
    </node>
</launch>
